docs/src/pages/components/data-grid/demo/FullFeauredDemo.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import { XGrid, GridToolbar } from '@material-ui/x-grid';
+import {
+  useDemoData,
+  getCommodityColumns,
+  getEmployeeColumns,
+  getRealData,
+} from '@material-ui/x-grid-data-generator';
+import KeyboardArrowRightIcon from '@material-ui/icons/KeyboardArrowRight';
+import { makeStyles } from '@material-ui/core/styles';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import Button from '@material-ui/core/Button';
+import FormLabel from '@material-ui/core/FormLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: 400,
+    width: '100%',
+    '& .MuiFormGroup-options': {
+      alignItems: 'center',
+      justifyContent: 'center',
```

Comment:
It would look better, I think without centering. Please let me know what you think about it. 
```suggestion
      marginTop: theme.spacing(2),
```

<img width="636" alt="Screenshot 2021-03-02 at 20 32 33" src="https://user-images.githubusercontent.com/3165635/109704055-74cf8500-7b96-11eb-970b-dc097ce36be7.png">